src/v2-cloud/meta-merger/meta-merger.test.ts

```
@@ -0,0 +1,245 @@
+
+function sortCRDTEntries(rows: CRDTEntry[]) {
+  return rows.sort((a, b) => a.cid.localeCompare(b.cid));
+}
+
+interface MetaConnection {
+  readonly metas: CRDTEntry[];
+  readonly connection: Connection;
+}
+
+function toCRDTEntries(rows: MetaConnection[]) {
+  return rows.reduce((r, i) => [...r, ...i.metas], [] as CRDTEntry[]);
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)